src/App/App.tsx

```diff
@@ -3,6 +3,7 @@ import {PlanetBio, AllData} from '../interface';
 import {discoverPlanets} from '../apiCalls.js';
 import planetData from '../data/planetData.js';
```

Comment:
This is a comment on previous code (not new in this PR) but I just noticed these `.js` file extensions. I think we don't need them, because I think `.js` is the default if a file extension isn't specified.